docs/admin.md

```diff
@@ -0,0 +1,96 @@
+# Administrator Guide
+
+## Backstage Operator configuration
+
+### Context
+
+As it is described in Design doc (TODO), Backstage CR's desired state is defined using layered configuration approach, which means:
+- By default each newly created Backstage CR uses Operator scope Default Configuration
+- Which can be fully or partially overriden for particular CR instance using ConfigMap with the name pointed in BackstageCR.spec.RawConfig 
+- Which in turn can be customized by other BackstageCR.spec fields (see Backstage API doc)
+
+Cluster Administrator may want to customize Default Configuration due to internal preferences/limitations, for example:
+- Preferences/restrictions for Backstage and|or PostgreSQL images due to  Airgapped environment.
```

Comment:
```suggestion
- Preferences or restrictions for Backstage;
- A PostgreSQL image that is available from within a restricted (air-gapped) environment;
```

Reply:
What I am trying to say is the  Backstage and|or PostgreSQL IMAGES can be a subject of customization of Backstage config.

Reply:
ok, revised my suggestion. is that closer to what you were trying to convey?